### PR TITLE
Pull changes from LarryEightBallSim

### PR DESCRIPTION
**Pull request by LQ1234**

### CODE REVIEW

**Review by prensing — APPROVED** [review]
Looks good to me, but wait on Coach Fitz.

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -182,10 +193,11 @@ public void teleopInit() {
     //SmartDashboard.putData(m_robotContainer.testFlup);
     if (m_autonomousCommand != null)
       m_autonomousCommand.cancel();
-    m_robotContainer.driveCommand.schedule();
+
+    //m_robotContainer.driveCommand.schedule();
```
Comment (coachFitz0807) [inline src/main/java/frc/robot/Robot.java]: Why did we comment this out? If this ever goes ontp the robot we want the drive command scheduled in teleopInit.

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -182,10 +193,11 @@ public void teleopInit() {
...
     //m_robotContainer.testFlup.schedule();
     //m_robotContainer.shooter.testSpin();
-    m_robotContainer.carouselCommand.schedule();
+    //m_robotContainer.carouselCommand.schedule();
```
Comment (coachFitz0807) [inline src/main/java/frc/robot/Robot.java]: Same comment. Don't we want to start the carousel too?

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (prensing) [inline src/main/java/frc/robot/Robot.java]: These should be undone. We want to match master/infiniteSimulator as much as possible

**Review by prensing — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (LQ1234) [reply src/main/java/frc/robot/Robot.java]: It is done

**Review by coachFitz0807 — APPROVED** [review]

**Review by prensing — APPROVED** [review]

**Pull request merged**